retrofit/src/main/java/retrofit2/Platform.java

```diff
@@ -32,12 +32,14 @@ static Platform get() {
   }
 
   private static Platform findPlatform() {
-    try {
-      Class.forName("android.os.Build");
-      if (Build.VERSION.SDK_INT != 0) {
-        return new Android();
+    if ("Dalvik".equals(System.getProperty("java.vm.name"))) {
```

Comment:
I am concerned that attempting to load any Android class runs the risk of undesirable consequences especially in an IDE environment. Case is point is the android.os.Builder class was not on the classpath in the IDEA 2016.2 version of the Android Support plugin but has appeared on the classpath in the 2016.3 version. In other words, it is impossible to predict what Google and JetBrains will expose either intentionally or accidentally. A property-based approach would be the safest.